scripts/chandra/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Create output dir
+mkdir ./output_files
+
+# First download the spectral data
+if python download_script.py 9000 80 1 catalog ./output_files/ ; then
+    echo "Build parent sample for deep field successful"
+else
+    echo "Build parent sample for deep field failed"
+    exit 1
+fi
+
+# Untar files
+for file in ./output_files/*.tar; do
+    tar -xvf "$file" -C ./output_files/
+done
+
+
+# Now build the parent sample for the deep field
+if python build_parent_sample.py catalog_hdf5.hdf5 parent_sample_xray.hdf5 ./output_files/ ; then
+    echo "Build parent sample for deep field successful"
+else
+    echo "Build parent sample for deep field failed"
+    exit 1
+fi
+
+# Try to load the dataset with hugging face dataset
+if python -c "from datasets import load_dataset; dset = load_dataset('./chandra.py', 'csc21_spectra', trust_remote_code=True, split='train').with_format('numpy'); next(iter(dset));"; then
+    echo "Load dataset for deep field successful"
+else
+    echo "Load dataset for deep field failed"
+    exit 1
+fi
```

Comment:
This suggestion is to allow everything to be more readable and to keep consistent with the argparse changes.

```suggestion
#!/bin/bash
DATA_PATH="./output_data"
# Create output dir
mkdir -p $DATA_PATH

# First download the spectral data
if python download_script.py --min_cnts 9000 --min_sig 80 --max_theta 1 --output_file catalog --file_path $DATA_PATH/ ; then
    echo "Download parent sample for deep field successful"
else
    echo "Download parent sample for Chandra spectra"
    exit 1
fi

# Untar files
for file in $DATA_PATH/*.tar; do
    tar -xvf "$file" -C $DATA_PATH/
done


# Now build the parent sample
if python build_parent_sample.py --cat_file catalog.hdf5 --output_file parent_sample_xray.hdf5 --file_path $DATA_PATH/  ; then
    echo "Build parent sample for deep field successful"
else
    echo "Build parent sample for Chandra spectra"
    exit 1
fi

# Try to load the dataset with huggingface dataset
if python -c "from datasets import load_dataset; dset = load_dataset('./chandra.py', 'csc21_spectra', trust_remote_code=True, split='train').with_format('numpy'); next(iter(dset));"; then
    echo "Load dataset for deep field successful"
else
    echo "Load dataset for deep field failed"
    exit 1
fi

```